tests/bluetooth/bsim_bt/bsim_test_eatt_notif/src/client_test.c

```diff
@@ -185,7 +202,10 @@ static void test_main(void)
 		k_sleep(K_TICKS(1));
 	}
 
+	printk("Subscribing client\n");
 	local_attr = &g_svc.attrs[1];
+	subscribe_client(g_conn, local_attr);
```

Comment:
This is purely a local operation on the server, right? 

Why not actually use the remote client to subscribe (properly) instead?

Reply:
Yes. Doing it the proper way distracts a bit from the test imo because of all the boilerplate code to discover the proper way (I don't want to hardcode server handles on the client). We could change that when the discovery helpers that you suggested are merged.

Reply:
> Yes. Doing it the proper way distracts a bit from the test imo because of all the boilerplate code to discover the proper way (I don't want to hardcode server handles on the client). We could change that when the discovery helpers that you suggested are merged.

True, but if the client doesn't subscribe properly, the notification is likely going to be discarded on the client side, right? So wouldn't that cause issues if we expand this test later?

Reply:
For now it isn't, but if we decide to actually do the subscription check in the client, then yeah. I'll try to do it properly.